linera-execution/src/lib.rs

```diff
@@ -52,9 +42,20 @@ use linera_base::{
     ownership::ChainOwnership,
 };
 use linera_views::{batch::Batch, views::ViewError};
+pub use policy::ResourceControlPolicy;
+pub use resources::{ResourceController, ResourceTracker};
 use serde::{Deserialize, Serialize};
-use std::{fmt, str::FromStr, sync::Arc};
+pub use system::{
+    SystemExecutionError, SystemExecutionStateView, SystemMessage, SystemOperation, SystemQuery,
+    SystemResponse,
+};
 use thiserror::Error;
+#[cfg(all(with_testing, any(with_wasmer, with_wasmtime)))]
```

Comment:
If we apply the previous suggestion, we could also fix this :sweat_smile: 

```suggestion
#[cfg(all(with_testing, with_wasm_runtime))]
```